docs/source/molecular_generation.rst

```diff
@@ -0,0 +1,58 @@
+.. generative_design_execution:
+
+[ALPHA] Generative Design Execution
+===================================
+The Generative Design Execution tool provided by the Citrine Platform is a feature that enables the creation of new molecules by making mutations to a set of seed molecules.
+To use this feature, the user provides a set of starting molecules and filtering parameters through the use of the :class:`~citrine.informatics.generative_design.GenerativeDesignInput` class.
+The class defines the seed molecules to generate mutations from, the fingerprint type used to calculate the fingerprint similarity, the minimum fingerprint similarity between the seed and the mutated molecule, and the number of initial mutations that will be attempted per seed.
```

Comment:
Link "fingerprint similarity" to a webpage explaining the concept in more detail. Perhaps this could be the RDKit documentation page that explains the various fingerprint types